### PR TITLE
Update S-O version to next release cycle

### PR DESCRIPTION
**Pull request by dsimansk**
Per title. Sorry for the delay. 

/cc @pierDipi @matzew @ReToCode @skonto

### CODE REVIEW

File: olm-catalog/serverless-operator/Dockerfile
Comment (dsimansk) [inline olm-catalog/serverless-operator/Dockerfile:11]: @Kaustubh-pande does this label have to include 3 channels for 1.30? Something like `stable,stable-1.30,stable-1.29`?

File: olm-catalog/serverless-operator/project.yaml
Comment (dsimansk) [reply olm-catalog/serverless-operator/project.yaml:9]: Thanks @matzew! It seems so to me. /cc @mgencur

Comment (dsimansk) [reply olm-catalog/serverless-operator/project.yaml:9]: Should we remove the first entry and add new one? To keep the list reasonably long & curated for upgrade path test?

Comment (pierDipi) [inline olm-catalog/serverless-operator/project.yaml:9]: We need to follow the comment above!

/hold

Comment (dsimansk) [reply olm-catalog/serverless-operator/project.yaml:9]: https://github.com/openshift-knative/serverless-operator/pull/2073/commits/a0d144a07a9b00ca688b0b933e5f0954ec0a1c40

File: olm-catalog/serverless-operator/project.yaml
Comment (pierDipi) [inline olm-catalog/serverless-operator/project.yaml:18]: I'm not entirely sure on the channel changes (like should we replace stable-1.29 or add stable-1.30?), so I'd leave that to others more familiar with the new release process or maybe you can describe why this would work?

File: olm-catalog/serverless-operator/project.yaml
Comment (pierDipi) [reply olm-catalog/serverless-operator/project.yaml:9]: For eventing, I'm pretty sure we can use the same format as `dependencies`

File: olm-catalog/serverless-operator/project.yaml
Comment (dsimansk) [reply olm-catalog/serverless-operator/project.yaml:18]: That'll add a label to the operator image. This's touching the same area from [my comment](https://github.com/openshift-knative/serverless-operator/pull/2073#discussion_r1193450198).

Per our slack chat, we can follow-up on the channel names in another PR. And get this merged to unblock operand bumps.

Comment (dsimansk) [reply olm-catalog/serverless-operator/project.yaml:18]: I.e.: adding another entry produces the expected labels:
```
diff --git a/olm-catalog/serverless-operator/metadata/annotations.yaml b/olm-catalog/serverless-operator/metadata/annotations.yaml
index 552c66acb..46aa81b07 100644
--- a/olm-catalog/serverless-operator/metadata/annotations.yaml
+++ b/olm-catalog/serverless-operator/metadata/annotations.yaml
@@ -1,6 +1,6 @@
 annotations:
   operators.operatorframework.io.bundle.channel.default.v1: stable
-  operators.operatorframework.io.bundle.channels.v1: stable,stable-1.30
+  operators.operatorframework.io.bundle.channels.v1: stable,stable-1.30,stable-1.29
   operators.operatorframework.io.bundle.manifests.v1: manifests/
   operators.operatorframework.io.bundle.mediatype.v1: registry+v1
   operators.operatorframework.io.bundle.metadata.v1: metadata/
diff --git a/olm-catalog/serverless-operator/project.yaml b/olm-catalog/serverless-operator/project.yaml
index 8c7da92d7..d1209c5a5 100644
--- a/olm-catalog/serverless-operator/project.yaml
+++ b/olm-catalog/serverless-operator/project.yaml
@@ -16,6 +16,7 @@ olm:
     list:
       - 'stable'
       - 'stable-1.30'
+      - 'stable-1.29'
   previous:
     replaces: 1.28.0
     skipRange: '>=1.28.0 <1.29.0'
```

File: olm-catalog/serverless-operator/project.yaml
Comment (dsimansk) [reply olm-catalog/serverless-operator/project.yaml:9]: Gotcha it now, thanks for more hints on this one. I see Serving is on  the format as Eventing components, I hope it works.

File: olm-catalog/serverless-operator/project.yaml
Comment (matzew) [inline olm-catalog/serverless-operator/project.yaml:67]: should these be `1.8.0` ? 
(by looking at the prev. two PRs for this exercise?)

Comment (dsimansk) [reply olm-catalog/serverless-operator/project.yaml:67]: I think Serving is on the same pattern from 1.8 https://github.com/openshift/release/blob/master/ci-operator/config/openshift-knative/serving/openshift-knative-serving-release-v1.8__412.yaml#L166-L170.

Comment (ReToCode) [reply olm-catalog/serverless-operator/project.yaml:67]: Yes, we are on the same pattern as Eventing now. Should be fine here too.

**Review by matzew — APPROVED** [review]
/lgtm
/approve

**Comment by matzew** [comment]
/hold

should close this, and apply the tool and have the tool do the job instead ?

**Comment by matzew** [comment]
This PR was great to again analyse the hassles. But if we do now merge this manual PR, we IMO do miss a chance to test the tool already for 1.30

**Comment by dsimansk** [comment]
This PR is superseded by #2078, closing.

/close

**Pull request closed without merging**
